src/main/java/org/sunbird/user/registration/service/UserRegistrationServiceImpl.java

```diff
@@ -288,9 +284,6 @@ private String validateRegisterationPayload(UserRegistrationInfo userRegInfo) {
 		if (StringUtils.isBlank(userRegInfo.getSource())) {
 			errList.add("Source");
 		}
-		if(StringUtils.isBlank(userRegInfo.getPhone())) {
-			errList.add("Phone");
-		}
```

Comment:
why this condition is removed - this is going to affect direct user registration flow.